examples/linux64/hello-make/Makefile

```diff
@@ -0,0 +1,16 @@
+CC = python -m ppci cc
+CFLAGS = -O1
+
+all: hello
+
+# Now link all compiled objects (here, just one) into executable. For this,
+# we need to provide linker with memory layout file. We'll also use main()
+# as the executable entrypoint.
+hello: hello.o
```

Comment:
I tried to put the `linux64.ld` file as dependency as well (since it is a dependency), but that did not work due to the nature of the `$^` variable.

Reply:
> I tried to put the linux64.ld file as dependency as well (since it is a dependency), but that did not work due to the nature of the $^ variable.

Yep. You either need to get rid of special vars, using "materialized" file list, or so-called order-only dependencies might help, but few people know about them, so using them might be confusing. This is again to show simple, clear, concise usage.